### ISSUE TITLE
mysql support??

### ISSUE THREAD

**Issue by BLAST1718** [Wertzui123/BedrockClans#24]

**Comment by Wertzui123** [comment]
No, MySQL is not supported by BedrockClans right now. Please put a little bit more effort into creating issues next time.  :wink:

**Comment by BLAST1718** [comment]
who kicked/banned me on ur discord server?

THE BLAST DRAGO#9255

**Comment by Wertzui123** [comment]
I have banned you for violation of rule 8:
Don't ping random people or developers when you need help - be patient

This is not an issue with BedrockClans though, so please contact me via dm next time.

**Comment by BLAST1718** [comment]
whats ur discord?

**Comment by DerDevHD** [comment]
> whats ur discord?

https://discord.gg/ZZ3DP8naXZ

**Comment by BLAST1718** [comment]
im banned there

**Comment by Wertzui123** [comment]
Wertzui123#1317

**Comment by efor89** [comment]
with mysql support i think it will be the best clan plugin
